flink/src/test/java/org/apache/iceberg/flink/TestFlinkSchemaUtil.java

```diff
@@ -313,6 +314,7 @@ public void testConvertFlinkSchemaWithPrimaryKeys() {
   }
 
   @Test
+  @Ignore
```

Comment:
not sure this test relevant anymore

Reply:
why not? this test looks legit to me

Reply:
fixed the test now, (didn't know nested field could be identifier field)